From 0.17.0 The custom scalars does'nt seem to work with interfaceType

After upgrade to `0.17.0` or `0.18.0` my custom scalars does not work.

> [ERROR] 10:06:55 TypeError: t.datetime is not a function

My scalars
```ts
import {asNexusMethod} from '@nexus/schema';
import {DateTimeResolver} from 'graphql-scalars';

export const GQLDateTime = asNexusMethod(DateTimeResolver, 'datetime');
```

And schema

```ts
import * as schema from '@nexus/schema';

export const CreatedAt = schema.interfaceType({
  name: 'CreatedAt',
  definition(t) {
    // Does'nt works
    t.datetime('createdAt', {
      resolve(field) {
        return new Date(field.created_at) as any;
      },
    });
    t.resolveType(() => null);
  },
});

export const User = schema.objectType({
  name: 'User',
  definition(t) {
    t.implements('CreatedAt');
    t.id('id');
    t.datetime('last_login_at'); // It Works
  },
});

```

Reproduction https://github.com/nghiepit/nexus-scalar-bug
If downgrade to `0.16.0` it work again.
Maybe related https://github.com/graphql-nexus/schema/issues/565

--- a/src/builder.ts
+++ b/src/builder.ts
@@ -809,7 +809,7 @@ export class SchemaBuilder {
           }),
         addModification: () => {},
         addField: () => {},
-        addDynamicOutputMembers: () => {},
+        addDynamicOutputMembers: (block, wrapping) => this.addDynamicOutputMembers(block, 'walk', wrapping),
         warn: () => {},
       })
       obj.definition(definitionBlock)

--- a/tests/interfaceType.spec.ts
+++ b/tests/interfaceType.spec.ts
@@ -1,6 +1,7 @@
 import { graphql } from 'graphql'
 import path from 'path'
-import { generateSchema, interfaceType, makeSchema, objectType, queryField } from '../src/core'
+import { DateTimeResolver } from 'graphql-scalars'
+import { generateSchema, interfaceType, makeSchema, objectType, queryField, asNexusMethod } from '../src/core'
 
 describe('interfaceType', () => {
   it('can be implemented by object types', async () => {
@@ -10,8 +11,11 @@ describe('interfaceType', () => {
           name: 'Node',
           definition(t) {
             t.id('id')
+            // @ts-ignore
+            t.dateTime('createdAt')
           },
         }),
+        asNexusMethod(DateTimeResolver, 'dateTime'),
         objectType({
           name: 'User',
           isTypeOf(data) {

